src/infra/usecases/xmldom-to-mathml-elements/error-handler.ts

```diff
@@ -21,15 +21,18 @@ export class ErrorHandler {
     const missingAttribute = errorMessage.split('"')[1];
     if (missingAttribute) return xml.replace(this._matchMissingValueForAttribute(missingAttribute), '');
 
-    return xml.replace(this._mathGenericMissingValue(), '');
+    while (this._mathGenericMissingValue().exec(xml)) {
```

Comment:
yes, it is, because it could happen that one tag has more than one attribute broken, something like:

```
<root>
<math>
<mfenced open= close= >
  <mn>3</mn>
</mfenced>
<mfenced open= close= >
  <mn>5</mn>
</mfenced>
</math>
</root>
```

Reply:
Ok. Understood. Thanks for the explanation.